airbyte-integrations/connectors/source-freshdesk/unit_tests/test_streams.py

```diff
@@ -1,211 +0,0 @@
-#
-# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
-#
-
-import random
-from typing import Any, MutableMapping
-from unittest.mock import PropertyMock
-
-import pytest
-from airbyte_cdk.models import SyncMode
-from airbyte_cdk.sources.streams import Stream
-from source_freshdesk.streams import (
-    Agents,
-    BusinessHours,
-    CannedResponseFolders,
-    CannedResponses,
-    Companies,
-    Contacts,
-    Conversations,
-    DiscussionCategories,
-    DiscussionComments,
-    DiscussionForums,
-    DiscussionTopics,
-    EmailConfigs,
-    EmailMailboxes,
-    Groups,
-    Products,
-    Roles,
-    SatisfactionRatings,
-    ScenarioAutomations,
-    Settings,
-    Skills,
-    SlaPolicies,
-    SolutionArticles,
-    SolutionCategories,
-    SolutionFolders,
-    Surveys,
-    TicketFields,
-    Tickets,
-    TimeEntries,
-)
-
-
-@pytest.fixture(autouse=True)
-def mock_tickets_use_cache(mocker):
-    mocker.patch("source_freshdesk.streams.Tickets.use_cache", new_callable=PropertyMock, return_value=False)
-
-
-def _read_full_refresh(stream_instance: Stream):
-    records = []
-    slices = stream_instance.stream_slices(sync_mode=SyncMode.full_refresh)
-    for slice in slices:
-        records.extend(list(stream_instance.read_records(stream_slice=slice, sync_mode=SyncMode.full_refresh)))
-    return records
-
-
-def _read_incremental(stream_instance: Stream, stream_state: MutableMapping[str, Any]):
-    res = []
-    slices = stream_instance.stream_slices(sync_mode=SyncMode.incremental, stream_state=stream_state)
-    for slice in slices:
-        records = stream_instance.read_records(sync_mode=SyncMode.incremental, stream_slice=slice, stream_state=stream_state)
-        for record in records:
-            res.append(record)
-    return res, stream_instance.state
-
-
-@pytest.mark.parametrize(
-    "stream, resource",
-    [
-        (Agents, "agents"),
-        (Companies, "companies"),
-        (Contacts, "contacts"),
-        (Groups, "groups"),
-        (Roles, "roles"),
-        (Skills, "skills"),
-        (TimeEntries, "time_entries"),
-        (SatisfactionRatings, "surveys/satisfaction_ratings"),
-        (BusinessHours, "business_hours"),
-        (CannedResponseFolders, "canned_response_folders"),
-        (DiscussionCategories, "discussions/categories"),
-        (EmailConfigs, "email_configs"),
-        (EmailMailboxes, "email/mailboxes"),
-        (Products, "products"),
-        (ScenarioAutomations, "scenario_automations"),
-        (SlaPolicies, "sla_policies"),
-        (SolutionCategories, "solutions/categories"),
-        (TicketFields, "ticket_fields"),
-        (Surveys, "surveys"),
-    ],
-)
-def test_full_refresh(stream, resource, authenticator, config, requests_mock):
```

Comment:
These tests still seem valid and applicable to the current implementation. Why remove them?

Reply:
Fixed